tests/test_transforms.py

```diff
@@ -22,14 +22,48 @@
     "display.max_columns",
     20,
     "display.width",
-    300,
+    150,
     "display.max_colwidth",
     75,
     "display.precision",
     3,
 )
 
 
+def test_merge_duplicate_columns():
+    """
+    Tests that this:
+    >>> df
+    ...      a    a    a  b    b  c
+    ... 0  NaN  4.0  7.0  1  4.0  1
+    ... 1  2.0  NaN  8.0  2  5.0  2
+    ... 2  3.0  6.0  NaN  3  NaN  3
+
+    Gets transformed into this:
+    >>> transforms._merge_duplicate_named_columns(df.copy())
+    ... df2
+    ...      a    b  c
+    ... 0  7.0  4.0  1
+    ... 1  8.0  5.0  2
+    ... 2  6.0  3.0  3
+    """
+    df = pd.DataFrame(
```

Comment:
I think this is the desired behaviour, matching what I've understood as VEDA's duplicate column merging logic, but I'm not sure how to test this in VEDA itself, so it'd be great if you guys could confirm. Thanks!

Reply:
Thanks for your feedback @Antti-L.

> But does it cover cases where some of the (original) columns has e.g. bfg, which would render equivalent to b? Would it be recognized as a duplicate column with the other b's ?

Would you mind elaborating or giving an example of this case?  I'm not sure exactly what you mean by this.

Reply:
I wrote `b~f~g`, for example an attribute column that has tilde-separated qualifiers (indexes).  Not sure why the tildes disappeared in my post...  So, is the merging applied to duplicate columns before or after processing those tildes (sorry I could not see that directly in the changed code; I am not a python coder).  I would think it should be after the processing of the tildes?

Reply:
Ah right, thanks @Antti-L that makes more sense :)
@olejandro where do the tildes in column names get processed? I'm hoping it's before here...

Reply:
Actually it is done later on in `process_flexible_import_tables` and in `process_user_constraint_tables`.

This is not the only issue though. There may also be attribute aliases, which have different names, but same meaning as duplicated columns, so merging the columns too early may affect what gets overwritten.

Also, column renaming is done in `normalize_column_aliases` which may create duplicates.

My feeling is that we should distinguish 2 cases:
- column names that represent TIMES attributes. These should be converted into rows; rows below overwrite rows above. We do this already in  `process_flexible_import_tables` and in `process_user_constraint_tables`.
- column names that represent indices of TIMES attributes. We can handle them as proposed in this PR, but it should happen after `normalize_column_aliases`.

Additional info: `config.all_attributes` and `config.attr_aliases` are sets of TIMES attributes and their aliases respectively. Maybe duplicate columns could be checked against these sets and, if not in any of the sets, merged according to the procedure proposed in this PR?

Reply:
@SamRWest I've opened #203 that handles column names representing TIMES attributes. Any chance you could test whether it works for AusTIMES?